core/src/com/unciv/ui/victoryscreen/RankingManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.unciv.ui.victoryscreen
+
+import com.unciv.logic.civilization.CivilizationInfo
+import kotlin.math.roundToInt
+
+class RankingManager(private val civInfo: CivilizationInfo) {
+
+    fun getRanking(category: RankingType) : Int {
+        return when(category) {
+            RankingType.Population -> civInfo.cities.sumBy { it.population.population }
+            RankingType.CropYield -> civInfo.statsForNextTurn.food.roundToInt()
+            RankingType.Production -> civInfo.statsForNextTurn.production.roundToInt()
+            RankingType.Gold -> civInfo.gold
+            RankingType.Land -> civInfo.cities.sumBy { it.tiles.size }
+            RankingType.Force -> civInfo.units.sumBy { it.baseUnit.strength }
+            RankingType.Happiness -> civInfo.getHappiness()
+            RankingType.Technologies -> civInfo.tech.researchedTechnologies.size
+            RankingType.Culture -> civInfo.policies.storedCulture
```

Comment:
I don't know too much about the civInfo API, but does storedCulture also include unspent culture and culture? is that under statsForNextTurn?

Reply:
`storedCulture` is how much culture has been ever produced both spent and unspent. `statsForNextTurn` stores the current rate of culture production.
If we rate the most cultural civ, we should take the first one.

Reply:
Just double check, the `storedCulture` does not include the already spent culture, only unspent.